### ISSUE TITLE
Where to Propose Sessions for the Developer Meetings?

### ISSUE THREAD

**Issue by flyingzumwalt** [ipfs/conf#24]
Should we have people propose sessions in this repository or should we do that elsewhere?

Example: If I want to propose having an [Allies Workshop](http://geekfeminism.wikia.com/wiki/Allies_workshop), and @lidel wants to propose having a working session about crafting our asks for Browser Manufacturers, where should we propose them?

**Comment by mgoelzer** [comment]
Duplicate of #12

**Comment by diasdavid** [comment]
The call is being uploaded at https://youtu.be/5-dRO3iHwF8.

In summary, I proposed two things:

- For accepting submissions - Accepting talks through github issues/prs vs google forms create different dynamics. We should learn from other confs that experimented with both and decide what is best for us. From my past experiences, open submissions through Github lead to more engagement from the community but it doesn't translate necessarily to better quality of the content, a good curated conference has to do a lot of work in the back office of scouting, encouraging and helping amazing people to volunteer to speak. 

- For evaluating the proposed talks - Let's have a rubric on how to rank talks and let each curator do its review independently, once that is done, we sort and do a final last pass. Given that time is short, we should tune up the rubrik to enable us to operate in a short period of time (e.g if the rubrik has 10 fields to evaluate with different weights, it is going to make it hard to finish in 48 hours).

**Comment by flyingzumwalt** [comment]
I've started assembling the instructions for proposing sessions (using PRs) in https://github.com/ipfs/developer-meetings/blob/proposing-sessions/sessions/README.md

**Comment by meiqimichelle** [comment]
I like what @flyingzumwalt started and would like to merge, which would at least soft-launch the submissions process. If there are no blockers, I'll do this tomorrow (to give other time zones a chance to weigh in).

**Comment by meiqimichelle** [comment]
Actuallllyyyy -- this is done for the IPFS dev meeting, but not for libp2p. Re-assigning to @mgoelzer so he can re-use this format for libp2p.
